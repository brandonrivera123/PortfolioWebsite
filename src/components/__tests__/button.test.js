import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { Button } from "..";

describe("Button Smoke Test", () => {
  it("renders button component", () => {
    render(<Button name="Test" />);
    const buttonText = screen.getByText(/test/i);
    expect(buttonText).toBeInTheDocument();
  });
  it("matches snapshot", () => {
    const tree = renderer.create(<Button className="button" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
