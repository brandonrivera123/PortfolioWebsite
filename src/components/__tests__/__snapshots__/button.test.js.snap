// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Smoke Test matches snapshot 1`] = `
<button
  className="sc-bdvvtL eAGxvH button"
>
  <div
    className="sc-dkPtRN iycVBf"
    color="#fff"
    onMouseDown={[Function]}
  />
  <span
    className="sc-gsDKAQ dXYkjU"
  >
    button
  </span>
</button>
`;
